tests/LanguageServerRename/OffsetExtractor.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerRename\Tests;
+
+use Closure;
+
+class OffsetExtractor
```

Comment:
could create this with a static constructor e.g. `::create()` and then a fluent API `::create()->registerX()->registerY()`